### ISSUE TITLE
"Move Focus to Terminal" key should be also shown in Help / Accessibility / Focus

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6924]
In RStudio Server v1.3.959, you would find `Tools / Terminal / "Move Focus to Terminal"`.

This is closely related to focus movement and accessibility, so I suggest showing this item under `Help / Accessibility / Focus` as well.

**Comment by gtritchie** [comment]
The Focus Terminal command already exists in two places: (1) under the View menu and (2) under Tools / Terminal.

Focus Console and many others currently only exist under the View menu. Not sure we should duplicate all of these under Help / Accessibility / Focus, but if we were going to do that for Terminal it seems we would want to do all of them.

Originally I was thinking Help / Accessibility / Focus submenu was only for commands created specifically for screen-reader users, but no particular reason for that.

**Comment by jooyoungseo** [comment]
This is thought-provoking. We could list all of the focus-related keys, and discuss as to whether they would be often used for screen-reader users. I believe the terminal focus is one of the go-to shortcuts for keyboard users, in particular, for those who have heavily relied on CLI over GUI.
